pixl_dcmd/tests/test_tag_schemes.py

```diff
@@ -49,9 +50,9 @@ def test_merge_base_only_tags(base_only_tag_scheme):
             count_tags[key] = 1
 
     for key, values in count_tags.items():
-        assert (
-            values == 1
-        ), f"{key} is replicated please check config files to remove it"
+        pytest_check.equal(
+            values, 1, msg=f"{key} is replicated please check config files to remove it"
+        )
```

Comment:
ooh interesting, why the change?

Reply:
oh just so that if there are several duplicated tags, we will get a list of all of them from running the test once (previously the test would fail after the first failed assert, you'd remove the duplicated tag, then it might fail again on another tag)

Reply:
ah yes of course